docs/MANPAGE_SYNTAX.md

```diff
@@ -29,45 +29,51 @@ podman\-command - short description
 
-#### **--option**, **-o**
 
-OPTIONS can be put after the command in two different ways. Eather the long version with **--option** or as the short version **-o**. If there are two ways to write an OPTION they are separated by a comma. If there are 2 (two) versions of one command the long version is always shown in front.
+#### **--version**, **-v** [=**false** | *true*]
 
-#### **--exit**
+OPTIONS can be put after the command in two different ways. Eather the long version with **--option** or as the short version **-o**. If there are two ways to write an OPTION they are separated by a comma. If there are 2 (two) versions of one command the long version is always shown in front. If the OPTION is boolean, it is not requiered to set an argument. In this case the equal sign is put in brackets `[]` with the arguments, in difference to arguments that are not boolean as shown with **--answer**. The default argument that is shown for a boolean OPTION is given when the OPTION is not used in the command. If the OPTION is used in the command and no argument is specified, the OPTION will always become `true`.\
+Example: *IMPORTANT: This option is not available with the remote Podman client*
+
+#### **--exit** [=**true** | *false*]
 
 An example of an OPTION that has only one possible structure. Thus, it cannot be executed by the extension **-e**.
 
-#### **--answer**=, **-a**=**_active_** | *disable*
+#### **--answer**=, **-a**=**active** | *disable*
 
-OPTIONS that accept 2 possible arguments as inputs are shown above. If there is a default argument that is selected when no special input is made, it is shown in **_bold italics_**. It must always be ensured that the standard argument is in the first place after the OPTION. In this example, there are 2 (two) different versions to execute the command. Both versions of the OPTION have to be shown with the arguments behind them.
+An example for an OPTIONS that accept 2 possible arguments as inputs is shown above. If there is a default argument that is selected when the OPTION is not used in the command, it is shown in **bold**. It must always be ensured that the standard argument is in the first place after the OPTION. In this example, there are 2 (two) different versions to execute the command. Both versions of the OPTION have to be shown with the arguments behind them. The default is **active**.
```

Comment:
I can't parse this either. Maybe you mean "An example for an OPTION that accepts ..."?

Also, same comment about `2 (two)`.

Reply:
Grammar error on my side.